frontend/packages/component-library/src/formFieldWrapper/FormFieldWrapper.tsx

```diff
@@ -0,0 +1,102 @@
+import classNames from 'classnames';
+import {ReactNode, useMemo, HTMLAttributes} from 'react';
+
+import {ComponentSizeXSToL} from '@/common/types';
+import FontAwesomeV6Icon, {FontAwesomeV6IconProps} from '@/fontAwesomeV6Icon';
+
+import moduleStyles from './formFieldWrapper.module.scss';
+
+export interface FormFieldWrapperProps extends HTMLAttributes<HTMLDivElement> {
+  /** FormFieldWrapper children */
+  children?: ReactNode;
+  /** FormFieldWrapper color */
+  color?: 'black' | 'gray' | 'white';
+  /** FormFieldWrapper size */
+  size?: Exclude<ComponentSizeXSToL, 'xs'>;
+  /** FormFieldWrapper label */
+  label?: ReactNode;
+  /** FormFieldWrapper helper section */
+  helper?: Partial<{
+    icon: FontAwesomeV6IconProps;
+    text: ReactNode;
+  }>;
+  /** FormFieldWrapper error message */
+  error?: ReactNode;
```

Comment:
nit, but: why not helperMessage, erorrMessage? short names are nice, but they are not that descriptive